src/openrct2-ui/windows/TopToolbar.cpp

```diff
@@ -1700,7 +1700,7 @@ class TopToolbar final : public Window
 
         auto screenPos = sourceScreenPos;
         uint16_t maxPossibleHeight = ZoomLevel::max().ApplyTo(
-            std::numeric_limits<decltype(TileElement::BaseHeight)>::max() - 32);
+            std::numeric_limits<decltype(TileElement::BaseHeight)>::max() - 1);
```

Comment:
Is this subtraction really necessary? The scenery entry's height is already subtracted a few lines down. I guess it may still be needed because IIRC the max base height is a way the game marks elements to be removed. Not sure if that's still the case.